docs/05_references.md

```diff
@@ -1,3 +1,4 @@
 # References
 
 - Scripts created by the following tutorials in [Shutil Module in Python | GeeksForGeeks](https://www.geeksforgeeks.org/shutil-module-in-python/).
+- Also read, to learn more about [How to Delete File in Python? | Scaler Topics](https://www.scaler.com/topics/python/how-to-delete-file-in-python/)
```

Comment:
Thank you for adding this reference. The content is briefly mentioned in the other reference I added, but since removing a directory can be a bit tricky, it's good to have an additional reference.

Before I approve and merge, how about we change the following?:

**Original version**
> Also read, to learn more about [How to Delete File in Python? | Scaler Topics](https://www.scaler.com/topics/python/how-to-delete-file-in-python/)

**Revised version**
> To learn more about removing files and folders by using Python, see [How to Delete File in Python? | Scaler Topics](https://www.scaler.com/topics/python/how-to-delete-file-in-python/).